inception-project/inception
# #2813 - Recommender calling out to an ELG service

**What's in the PR**
- Added a very simple recommendr that can call out to an ELG service - no async support, not even error handling, but already can be used with some (fast) services

**How to test manually**
* Create an ELG recommender on a string feature of a span layer
* Sign in to ELG and fetch the success token the (the ELG session panel in the recommender settings guides you through it)
* Select a working ELG service, e.g. the GATE English Named Entity Recommender
* See if the recommender works on the annotation page

**Automatic testing**

**Documentation**
